package.json

```diff
@@ -28,7 +28,7 @@
     },
     "scripts": {
         "prepare": "tsc --outDir dist --declaration --declarationDir dist",
-        "test": "jest --ci",
+        "test": "jest",
```

Comment:
can we keep the ci flag? https://jestjs.io/docs/cli#--ci

Reply:
That slipped in after my local testing, fixing it